cli/cli.go

```diff
@@ -0,0 +1,24 @@
+package cli
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/docker/docker/pkg/homedir"
+)
+
+var (
+	configDir     = os.Getenv("DOCKER_CONFIG")
+	configFileDir = ".docker"
+)
+
+// ConfigurationDir returns the path to the configuration directory as specified by the DOCKER_CONFIG environment variable.
+func ConfigurationDir() string {
```

Comment:
@tiborvass I guess this is required for external code (like in `integration-cli`), right ? If true, could we had a `FIXME`/`TODO` for our future selves 👼